src/flb_log_event_encoder_primitives.c

```diff
@@ -519,8 +519,8 @@ int flb_log_event_encoder_append_forward_v1_timestamp(
 {
     uint32_t value[2];
 
-    value[0] = FLB_BSWAP_32((uint32_t) timestamp->tm.tv_sec);
-    value[1] = FLB_BSWAP_32((uint32_t) timestamp->tm.tv_nsec);
+    value[0] = FLB_TO_NATIVE_UINT32((uint32_t) timestamp->tm.tv_sec);
```

Comment:
Please add  `FLB_UINT32_TO_NETWORK_BYTE_ORDER` and use it here, even if the underlying implementation is still a swap, the idea is to mimic `htons` and `ntohs` in a clear and easy to understand manner.

Reply:
Please don't forget about this one, it's not outdated.

Reply:
Could you please clarify your suggestion? I'm not sure where the `FLB_UINT32_TO_NETWORK_BYTE_ORDER` function should be added.

Is this the definition you had in mind?

```
#define FLB_UINT32_TO_NETWORK_BYTE_ORDER(x) htonl(x)
```

Reply:
In practical terms it's the same as `FLB_UINT32_TO_HOST_BYTE_ORDER` so you could even define a macro like `#define FLB_UINT32_TO_NETWORK_BYTE_ORDER(value) FLB_UINT32_TO_HOST_BYTE_ORDER(value)`.

The point is not the code it generates but how the user interprets it when they read the code.

Reply:
I know it could sound silly but making sure the code is very clear helps avoid misunderstandings in the future when someone else works in it.

Reply:
Ok, that seems reasonable. I've updated the code based on your suggestions. My only question is where to place the macro `FLB_UINT32_TO_NETWORK_BYTE_ORDER`? Currently it's in `src/flb_log_event_encoder_primitives.c`. It's difficult to me to decide since I'm not fully familiar with your code base.